user_data/strategies/test_strategy.py

```diff
@@ -119,6 +121,16 @@ def populate_indicators(self, dataframe: DataFrame) -> DataFrame:
         # Overlap Studies
         # ------------------------------------
 
+
+        # Previous Bollinger bands
+        # Because ta.BBANDS implementation is broken with small numbers, it actually
+        # returns middle band for all the three bands. Switch to qtpylib.bollinger_bands
+        # and use middle band instead.
+        # Is broken
+        """
+        dataframe['blower'] = ta.BBANDS(dataframe, nbdevup=2, nbdevdn=2)['lowerband']
+        """
```

Comment:
will this fix #533, which also states that the wrong buy signals are used (in relation to Bollinger bands)

Reply:
I believe so, as, pandas no longer likes the values being put into blower otherwise it returns nan as a key error.